app/helpers/policies_helper.rb

```diff
@@ -9,6 +9,33 @@ def policy_profile_from_scap_content(policy)
     policy.scap_content_profile.nil? ? "Default" : policy.scap_content_profile.title
   end
 
+  def deploy_by_radios(f, policy)
+    ForemanOpenscap::ConfigNameService.new.configs.map do |tool|
```

Comment:
options should either have inline help with some explanation what to expect or the tab should have generic help text explaining that puppet requires puppet agent configured and respective puppet class will be assigned to the host/hostgroup, similarly for ansible where user should be warned about SSL cert setup. For manual, user should be told about what are the steps to be taken for successful deployment.

Reply:
I added inline notification with link to docs.